apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -41,38 +40,42 @@ <h1>View Users</h1>
                   <td>{{r.user_email}}</td>
                   <td>{{r.user_name}}</td>
                   <td>{{r.org_name}}</td>
-                  <td>{{r.role_id}}</td>
+                  <td>{{r.role_name}}</td>
+                  <td>{{r.active}}</td>
                   <td>{{r.created_at}}</td>
                   <td>{{r.updated_at}}</td>
-                  <td>
-                    {{r.notes}}
-                  </td>
+                  <td>{{r.notes}}</td>
               </tr>
           {% endfor %}
       </tbody>
   </table>
 </div>
 <script>
+/*filters by organization with dropdown*/
```

Comment:
UnitedHealthcare (UHC) not pulling up when selecting from the dropdown "UnitedHealthcare (UHC)". UnitedHealthcare (UHC) only shows up under UHC which might have something to do with there being no spaces between UnitedHealthcare ??

Reply:
Hmm. Weird. Current page shows the UnitedHealthcare (UHC) when dropdown chosen. Shouldn't have changed anything there.